.github/workflows/lawa-ruby-ci.yml

```diff
@@ -18,6 +18,8 @@ on:
     secrets:
       github-token:
         required: true
+      sidekiq-enterprise-config:
```

Comment:
Can we make this a bit more generic to allow passing any bundler config? I don't we need to couple this action to sidekiq directly

Reply:
I see, something like pass it the entire config i.e. setting the environment variable? Rather than the value to set?

Reply:
i think that would make it a lot more reusable. maybe something similiar like you suggested earlier for the env and pass `custom-bundler-config` as a secret

Reply:
The reason I bring this up is because, we have to set a specific environment variable/bundle config. If the value we pass is generic, I am not sure how would we infer what environment variable/bundle config we need to set. Although I do agree, if this can be generalised, will be better, I will see if that is doable as well.

Reply:
Tested a few approaches but 

- neither[ declaring `env` globally in licensee check action](https://github.com/phrase/phrase/pull/11742/commits/0362b3eae0adcf5a5f78e916462ce9300243e00b) in phrase repo helps
- also since we are using `bundler-cache: true` option, as per the documentation, we can only set bundle config by means [of env variables](https://github.com/ruby/setup-ruby#bundle-config)